packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -557,4 +692,20 @@ export interface Borders {
   warning: Border;
 }
 
-export const defaultTheme = createThemeAdaptor(defaultThemeOptions);
+/**
+ * Default theme, uses light mode with no option to change it.
+ *
+ * @deprecated Use the `theme` to get a flexible light/dark mode theme function
+ */
+export const defaultTheme = createThemeAdaptor(
+  makeThemeOptions(chooseTheme("light"), "light")
```

Comment:
feels like potentially `defaultTheme()` could call `theme()` under the hood?

Something like:

```ts
export const defaultTheme = theme("light");
```

What do you think 🙆‍♂️ ?